### PR TITLE
User profile page

### PR DESCRIPTION
**Pull request by muktha-s**
made a list of the user's favorite restaurants and favorite categories appear on the front end

### CODE REVIEW

**Comment by aayush110** [comment]
LGTM Muktha! I can create another PR and further tweak this. But as of now, I think we can merge!

File: pages/index.tsx [hunk 1]
```diff
@@ -59,19 +59,13 @@ export default function Home({restaurants}: HomeProps) {
 import * as DataService from '@/lib/DataService'
 import Restaurant from '@/models/Restaurant';
 
-/**
- * Loads data on the server side as the page loads.
- * @returns properties for the `index` page with the data loaded.
- */
 export async function getStaticProps() {
```
Comment (ajaygandecha) [inline pages/index.tsx:88]: I would re-add this deleted documentation!

**Review by ajaygandecha — CHANGES_REQUESTED** [review]
Just adding a comment here to restore the previous documentation before we merge - otherwise, looks great!

File: pages/index.tsx (hunk 1 above)
Comment (ajaygandecha) [reply pages/index.tsx:88]: Maybe edit the `index` part to `user/[id]` so that the documentation is accurate!

**Review by ajaygandecha — APPROVED** [review]
LGTM! Ready to merge.

**Comment by ajaygandecha** [comment]
Let me know when this needs a re-review @aayush110 and @muktha-s .

**Pull request merged**